### DISCUSSION TITLE
Can I predict elastic constant through this?

### DISCUSSION THREAD

**Discussion by tngksldia** [janosh/matbench-discovery#79] [Q&A] [answered]
How can I import the trained model to predict elastic constant. I want the model trained from m3gnet, chgnet, mace. MEGNET provides the trained model itself. Though, I cannot find the trained model for others.

**Comment by janosh** [comment, marked as answer]
all models you list will auto-download their respective checkpoints when you install the corresponding PyPI package and run the model. see e.g. [`CHGNet.load()`](https://chgnet.lbl.gov/#usage) or [`mace_mp()`](https://mace-docs.readthedocs.io/en/latest/examples/foundation_models.html#pretrained-mace-mp-0-models).

**Reply by tngksldia** [reply]
Thank you for your consideration! Though, I wonder how can I get 'machine learned Elastic Constant', which can not be directly got from the machine learned potential.

**Reply by janosh** [reply]
have a look at the [`ElasticityCalc`](https://github.com/materialsvirtuallab/matcalc/blob/bbd67fff5a6822088b925640becdf8fb281a38dc/matcalc/elasticity.py#L22) class in [`matcalc`](https://github.com/materialsvirtuallab/matcalc)

**Reply by JaGeo** [reply]
Or the elastic workflow in atomate2: 
https://jageo.github.io/TutorialAtomate2Forcefields/intro.html
😃

**Reply by tngksldia** [reply]
Thanks everyone! I'll try it!
